ddtrace/utils/http.py

```diff
@@ -1,27 +1,10 @@
-from typing import Optional
+from ..internal.utils.deprecation import deprecation
+from ..internal.utils.http import normalize_header_name  # noqa
+from ..internal.utils.http import strip_query_string  # noqa
 
 
-def normalize_header_name(header_name):
-    # type: (Optional[str]) -> Optional[str]
-    """
-    Normalizes an header name to lower case, stripping all its leading and trailing white spaces.
-    :param header_name: the header name to normalize
-    :type header_name: str
-    :return: the normalized header name
-    :rtype: str
-    """
-    return header_name.strip().lower() if header_name is not None else None
-
-
-def strip_query_string(url):
-    # type: (str) -> str
-    """
-    Strips the query string from a URL for use as tag in spans.
-    :param url: The URL to be stripped
-    :return: The given URL without query strings
-    """
-    hqs, fs, f = url.partition("#")
-    h, _, _ = hqs.partition("?")
-    if not f:
-        return h
-    return h + fs + f
+deprecation(
+    name="ddtrace.utils.http",
+    message="Use `ddtrace.internal.utils.http` module instead",
```

Comment:
There are a few more of these still that need to be changed over

Reply:
@Kyle-Verhoog  - true, my oversight. I changed all of the deprecation messages accordingly. I did not change the message in ddtrace/contrib/util.py because I did not write the original message.